app/mailers/user_mailer.rb

```diff
@@ -21,4 +21,18 @@ def claim_submitted_notification(user, claim)
                          amount: claim.amount.format(symbol: true, decimal_mark: ".", no_cents: false),
                          link_to_claim:)
   end
+
+  def claim_submitted_support_notification(claim)
```

Comment:
```suggestion
  def claim_added_support_notification(claim)
```
Can we rename this? It's not actually submitting the claim